server/src/pogo-accounts.routes.ts

```
@@ -1,6 +1,7 @@
 import * as express from "express"
 import * as mongodb from "mongodb"
 import { collections } from "./database"
+import * as escape from "escape-html"
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)